google/provider.go

```diff
@@ -93,130 +93,127 @@ func Provider() terraform.ResourceProvider {
 			"google_compute_backend_service":         dataSourceGoogleComputeBackendService(),
 		},
 
-		ResourcesMap: map[string]*schema.Resource{
-			"google_bigquery_dataset":                      resourceBigQueryDataset(),
```

Comment:
The only different is that I removed the 6 resources in GeneratedComputeResourcesMap from this list